torch/_dynamo/symbolic_convert.py

```diff
@@ -2187,6 +2190,7 @@ def create_call_resume_at(self, inst):
         if new_code.co_freevars:
             cg.make_function_with_closure(name, new_code, True, stack_len)
         else:
+            # This is safe: we pre-generate a unique name
```

Comment:
Err, why? It looks to me like `__resume_at_20` could easily clobber other resumes?

Reply:
`unique_id` returns a unique number every time it is called. This number is never reset. `__resume_at_20` will only be generated once per python program.